suites/quincy/rbd/tier-2_rbd_namespace.yaml

```diff
@@ -0,0 +1,85 @@
+# Tier2: Added RBD namespace creation in default pool testing
```

Comment:
please make it as `Tier2: Test suite to cover RBD namespace related tests.`
while adding new test related to namespace use same suite.

Reply:
done